src/components/QuickOpenModal.js

```diff
@@ -398,6 +398,7 @@ export class QuickOpenModal extends Component<Props, State> {
           selectedItemId={
             expanded && items[selectedIndex] ? items[selectedIndex].id : ""
           }
+          {...(this.isSourceSearch() ? { size: "big" } : {})}
```

Comment:
I prefer the big here all the time as they're all one modal. But I agree with the change in project search.